### PR TITLE
Fixed update admin password sometimes sending false succeeded webhook

### PR DESCRIPTION
**Pull request by juanpardo**
## Summary of changes
Fixed update admin password sometimes sending false succeeded webhook

## Checklist

- [x] Relevant changes are reflected in `CHANGES.rst`
- [ ] Added or changed code is covered by tests
- [ ] Documentation has been updated if necessary
- [x] Changed code does not contain any breaking changes (or this is a major version change)

### CODE REVIEW

File: crate/operator/update_user_password.py [hunk 1]
```diff
@@ -104,9 +110,8 @@ async def update_user_password(
...
             raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
-        else:
-            if "ALTER OK" in result:
-                logger.info("... success")
-            else:
-                logger.info("... error. %s", result)
-                raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
+        except Exception as e:
+            exception_logger(
+                "... failed. Unexpected exception was raised. Class: %s",
```
Comment (SStorm) [inline crate/operator/update_user_password.py:115]: Might be worth also including the message?
```suggestion
                "... failed. Unexpected exception was raised. Class: %s, msg: %s", type(e).__name__, str(e)
```

File: crate/operator/update_user_password.py [hunk 2]
```diff
@@ -84,6 +84,12 @@ async def update_user_password(
                 stdout=True,
                 tty=False,
             )
+            if "ALTER OK" in result:
+                logger.info("... success")
+            else:
+                logger.info("... error. %s", result)
+                raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
```
Comment (SStorm) [inline crate/operator/update_user_password.py:91]: Won't this get caught in the `except Exception as e` handler below and then not propagated upwards?

Comment (juanpardo) [reply crate/operator/update_user_password.py:91]: Very good point!

File: crate/operator/update_user_password.py [hunk 3]
```diff
@@ -104,9 +110,12 @@ async def update_user_password(
...
             exception_logger("... failed. Status: %s Message: %s", e.status, e.message)
             raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
-        else:
-            if "ALTER OK" in result:
-                logger.info("... success")
-            else:
-                logger.info("... error. %s", result)
-                raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
+        except TemporaryError as e:
+            raise e
```
Comment (SStorm) [inline crate/operator/update_user_password.py:114]:
```suggestion
        except TemporaryError:
            raise
```

The exception is implicit ;-)

**Review by SStorm — APPROVED** [review]

**Pull request merged**